common/types.go

```diff
@@ -56,10 +56,11 @@ const (
 	DefaultDirectoryPermissionBits  os.FileMode = 0775
 	DefaultAllowOtherPermissionBits os.FileMode = 0777
 
-	MbToBytes  = 1024 * 1024
-	GbToBytes  = 1024 * 1024 * 1024
-	TbToBytes  = 1024 * 1024 * 1024 * 1024
-	CfuseStats = "cloudfuse_stats"
+	MbToBytes       = 1024 * 1024
+	GbToBytes       = 1024 * 1024 * 1024
+	TbToBytes       = 1024 * 1024 * 1024 * 1024
+	DefaultCapacity = TbToBytes
```

Comment:
I think it might be a bit cleaner to have this as DefaultCapacityMB and have it in MB here so we don't have to do the division to MB in case this changes in the future.

Reply:
Like this?
`DefaultCapacityMb = TbToBytes / MbToBytes`

Reply:
I think either is fine, but the first one might be a bit clearer that it is a terabyte.